Source/UI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -236,6 +236,7 @@ extension PaymentViewController {
     viewModel.updateCardNumberView = { [weak self] in
       DispatchQueue.main.async {
         self?.updateCardNumber()
+        self?.view.layoutSubviews()
```

Comment:
This looks like we're connecting the card number input change with some form of subview resizing. I am not sure that should be needed, what is the problem we are solving with this line?